src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -38,7 +38,7 @@ class LambdaTest extends AnyFlatSpec with Matchers with TestContainerUtils {
     val actual: APIGatewayProxyResponseEvent = lambda.process(input, client, config)
```

Comment:
Not really related to this PR but would it be possible to put in a unit test for the `Encode` object. Can test to the vals in the object:

- `alphabet`: check correct class / throws an error
- `baseNumber`: check correct base number (25) is used
- `referencePrefix`: check correct reference prefix (Z) is used